src/styles/index.scss

```diff
@@ -1,3 +1,83 @@
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  $minutes-diraction: 3600;
+  $seconds-diraction: 60s;
```

Comment:
There is a typo in the variable name `$minutes-diraction` which should be `$minutes-duration`. Variable names should clearly describe their purpose and follow the naming conventions.